build/travis/job2_AppImage/bintray.sh

```diff
@@ -261,8 +297,16 @@ if [ "${APPNAME}" == "MuseScoreNightly" ] && [ $(env | grep TRAVIS_JOB_ID ) ] ;
     }
   }'
   ${CURL} -X POST -d "${data}" ${API}/packages/${BINTRAY_REPO_OWNER}/${BINTRAY_REPO}/${PCK_NAME}/versions/${VERSION}/release_notes
+  echo ""
+  fi
+
+  # Delete older versions of non-release packages (nightlies and dev. builds)
+  HERE="$(dirname "$(readlink -f "${0}")")"
+  "${HERE}/bintray-tidy.sh" -s archive "${BINTRAY_REPO_OWNER}/${BINTRAY_REPO}/${PCK_NAME}"
```

Comment:
I'm still running a few tests. The `-s` (simulate) option needs to be removed from this line before it gets merged.